src/web/client/schema/portalSchema.ts

```diff
@@ -74,6 +74,8 @@ export const portal_schema_V1 = {
                 _languagegroupby: "adx_rootwebpageid",
                 _fetchQueryParameters:
                     "?$filter=adx_webpageid%20eq%20{entityId}&$select=adx_name,adx_copy,adx_customcss,adx_customjavascript,adx_partialurl,_adx_webpagelanguageid_value",
+                _multiFileFetchQueryParameters:
+                    "?$filter=_adx_websiteid_value%20eq%20{websiteId}&$select=adx_webpageid,_adx_webpagelanguageid_value,adx_name,adx_copy,adx_customcss,adx_customjavascript,adx_partialurl",
```

Comment:
This looks so error prone by hand crafting the space character encoding. 
Remove the encoding and let browser handle it.